client/post-editor/editor-ground-control/index.jsx

```diff
@@ -370,18 +291,14 @@ const EditorGroundControl = React.createClass( {
 						{ this.getPreviewLabel() }
 					</button>
 					<div className="editor-ground-control__publish-combo">
-						<button
-							className="editor-ground-control__publish-button button is-primary"
-							onClick={ this.onPrimaryButtonClick }
-							disabled={ ! this.isPrimaryButtonEnabled() }
+						<EditorPublishButton
+							{ ...this.props }
 							tabIndex={ 5 }
-						>
-							{ this.getPrimaryButtonLabel() }
-						</button>
-						{ siteUtils.userCan( 'publish_posts', this.props.site ) &&
+						/>
+						{ this.canSchedulePost() &&
 							<button
 								ref="schedulePost"
-								className="editor-ground-control__time-button button is-primary"
```

Comment:
I'm not hiding it. I'm hiding the big blue button ("Publish"), but the calendar button is just styled differently. I've tried just rendering 2 different versions of the button, one for mobile and one for desktop, and showing only one by CSS, but the calendar `Popover` anchor was messed up (because it has to anchor to a element by `ref`).